From fc16222abe4505238f4f20d211de1cc09a08c6d0 Mon Sep 17 00:00:00 2001
In-Reply-To: <20171227233912.GB181628@aiede.mtv.corp.google.com>
References: <20171227233912.GB181628@aiede.mtv.corp.google.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 27 Dec 2017 22:51:27 +0000
Subject: [PATCH v2 2/2] Windows: stop supplying BLK_SHA1=YesPlease by default
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Using BLK_SHA1 in lieu of the OpenSSL routines was done in
9bccfcdbff ("Windows: use BLK_SHA1 again", 2009-10-22), since DC_SHA1
is now the default for git in general it makes sense for Windows to
use that too, this looks like something that was missed back in
e6b07da278 ("Makefile: make DC_SHA1 the default", 2017-03-17).

As noted in 2cfc70f0de ("mingw: use OpenSSL's SHA-1 routines",
2017-02-09) OpenSSL has a performance benefit compared to BLK_SHA1 on
MinGW, so perhaps that and the Windows default should be changed
around again. That's a topic for another series, it seems clear that
this specific flag is nobody's explicit intention.

Reviewed-by: Jonathan Nieder <jrnieder@gmail.com>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 config.mak.uname | 1 -
 1 file changed, 1 deletion(-)

diff --git a/config.mak.uname b/config.mak.uname
index 685a80d138..6a862abd35 100644
--- a/config.mak.uname
+++ b/config.mak.uname
@@ -361,7 +361,6 @@ ifeq ($(uname_S),Windows)
 	NO_REGEX = YesPlease
 	NO_GETTEXT = YesPlease
 	NO_PYTHON = YesPlease
-	BLK_SHA1 = YesPlease
 	ETAGS_TARGET = ETAGS
 	NO_INET_PTON = YesPlease
 	NO_INET_NTOP = YesPlease
-- 
2.15.1.424.g9478a66081

